### PR TITLE
Configure node with relay address

### PR DESCRIPTION
**Pull request by sandreae**
Configure the app to accept `RELAY_ADDRESS` values via the command line `flutter run --dart-define=RELAY_ADDRESS=x.x.x.x:2022` or environment variables. This value is then passed into `startNode` and used in `relay_addresses` on network config. The `aquadoggo` running in your app should now connect to the configured relay and any other peers it discovers.

### CODE REVIEW

File: packages/app/lib/io/p2panda/node.dart [hunk 1]
```diff
@@ -6,6 +6,10 @@ import 'package:app/io/p2panda/key_pair.dart';
 import 'package:app/io/p2panda/p2panda.dart';
 import 'package:app/utils/sleep.dart';
 
+const List<String> relayAddresses = bool.hasEnvironment("RELAY_ADDRESS")
```
Comment (sandreae) [inline packages/app/lib/io/p2panda/node.dart:9]: I easily find a way to pass a list in via environment vars and I thought this was good enough for now.

File: packages/app/android/app/src/debug/AndroidManifest.xml [hunk 2]
```diff
@@ -3,5 +3,6 @@
          the Flutter tool needs it to communicate with the running application
          to allow setting breakpoints, to provide hot reload, etc.
     -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (adzialocha) [inline packages/app/android/app/src/debug/AndroidManifest.xml:7]: Do we need this permission?

**Review by adzialocha — CHANGES_REQUESTED** [review]
Think this needs documentation in  README.md or similar, otherwise looking good!

File: packages/app/lib/io/p2panda/node.dart (hunk 1 above)
Comment (adzialocha) [reply packages/app/lib/io/p2panda/node.dart:9]: Think this is a good solution!

File: packages/app/android/app/src/debug/AndroidManifest.xml (hunk 2 above)
Comment (sandreae) [reply packages/app/android/app/src/debug/AndroidManifest.xml]: nope!

**Pull request merged**